engine_test.go

```diff
@@ -203,7 +204,18 @@ steps:
       name: !expr $.input.name
 outputs:
   success:
-    message: !expr $.steps.example.outputs.success.message`),
+    message: !expr $.steps.example.outputs.success.message
+outputSchema:
+  success:
+    schema:
+      root: RootObjectOut
+      objects: 
+        RootObjectOut: 
+          id: RootObjectOut
+          properties:
+            message:
+              type:
+                type_id: string`),
```

Comment:
Intentional/incidental that you terminated the `TestE2E` workflow with a newline but didn't here? That is, the change in `TestE2E` suggests that the trailing newline is significant: but if so, why not here as well?

Reply:
I forgot to remove that newline. I pushed the change.